signer/src/bitcoin/validation.rs

```diff
@@ -249,6 +265,13 @@ impl BitcoinPreSignRequest {
             .await?
             .ok_or(Error::MissingSignerUtxo)?;
 
+        let bitcoin_client = ctx.get_bitcoin_client();
+
+        // Get the last fees from the mempool ourselves instead of using those provided by the
+        // the coordinator so that we assess whether the proposed fee is acceptable ourselves.
+        let mut signers_view_of_last_fees: Option<super::utxo::Fees> =
```

Comment:
nit: the `Fees` object is already imported.
```suggestion
        let mut signers_view_of_last_fees: Option<Fees> =
```

Reply:
Done.